### PR TITLE
Reorder production deployment docs

### PR DESCRIPTION
**Pull request by JohnGarbutt**

### CODE REVIEW

File: docs/best-practice.md [hunk 1]
```diff
@@ -117,3 +126,6 @@ a production deployment.
 You are now ready to begin adding configuration to your environments. When building an environment
 for the first time, it is recommended to follow each documentation page in order, beginning with
 the [Deployment method](./configuration/02-deployment-method.md).
+
+Remember to share as much configuration as possible between all your
+[environments](./environments.md)!
```
Comment (mkjpryor) [inline docs/best-practice.md:139]: Maybe make this a tip?

```
!!! tip

    Remember to share as much configuration as possible between all your [environments](./environments.md)!
```

File: docs/best-practice.md [hunk 2]
```diff
@@ -103,6 +112,14 @@ A
 [sample GitLab CI/CD configuration](https://github.com/stackhpc/azimuth-config/tree/stable/.gitlab-ci.yml.sample)
 is provided that implements this workflow for GitLab-hosted repositories.
 
+## Monitoring
+
+Before going into production, be sure you can
+[access the monitoring](./debugging/access-monitoring.md).
+Also ensure you have
```
Comment (mkjpryor) [inline docs/best-practice.md:119]: I think make these separate paragraphs for clarity.

**Review by mkjpryor — CHANGES_REQUESTED** [review]
I think this is a better first effort. Just a couple of minor tweaks.

File: docs/best-practice.md (hunk 1 above)
Comment (JohnGarbutt) [reply docs/best-practice.md:139]: feel free to add that, I have to run.

File: docs/best-practice.md [hunk 3]
```diff
@@ -83,6 +53,45 @@ base --> ha --> site --> site-ha --> production
...
+Azimuth creates application credentials to be able to create the
+user workloads using the appropriate automation.
+
+## Deployment Prerequisites
+
+Before proceeding with an Azimuth [deployment](./deployment/index.md),
+you should ensure that the target cloud
+meets the [prerequisites](./configuration/01-prerequisites.md).
+The prerequisite docs include details around the need for a wildcard DNS entry,
+matching TLS certificates and creating OpenStack application credentials.
```
Comment (mkjpryor) [inline docs/best-practice.md:82]: This wording I find slightly clunky. What do you think of something like:

> Before proceeding with an [Azimuth deployment](./deployment/index.md),
> you should ensure that you can fulfil all the [prerequisites](./configuration/01-prerequisites.md).
>
> This includes verifying that the target cloud has the required features available, generating
> credentials and ensuring that the required DNS records and TLS certificates are available.

**Review by mkjpryor — CHANGES_REQUESTED** [review]
Just one minor rewording, I think.

**Review by mkjpryor — APPROVED** [review]
LGTM

**Pull request merged**